jobrunner/job_executor.py

```diff
@@ -36,6 +36,7 @@ class JobDefinition:
     cpu_count: str = None  # number of CPUs to be allocated
     memory_limit: str = None  # memory limit to apply
     level4_max_filesize: int = 16 * 1024 * 1024
+    level4_max_csv_rows: int = 5000
```

Comment:
Do we need a default here?  I'm a little bit wary of having two places that look like they define what the limit is.  If somebody were to change this, then there'd be no effect since it gets set in `job_to_job_definition`.

Reply:
No, I think we can remove it. Your comment would also apply to the `level4_max_filesize` on the line above.

Reply:
The reason there is a default for filesize was mainly testing convenience, so you didn't have to specify it every time you instantiate a job_definition directly (which many executor tests do, as they can't use `job_to_job_defintion` in most cases). So, not having a default will probably require updating a bunch of test cases/factories

An alternative might be to default to `None`, and update the consuming code to handle that case, perhaps?

Reply:
It only involved updating one fixture and one function (in `test_local_executor.py` and `test_logging_executor.py` respectively), so i think this is fine.